.circleci/config.yml

```diff
@@ -32,12 +32,12 @@ jobs:
     docker:
       - image: circleci/node:8
     steps:
-      - attach_workspace:
-          at: .
```

Comment:
With this gone, you could drop the `persist_to_workspace` to workspace stuff above entirely, which should speed up the builds a little.

Reply:
Good point!